app/legacy_lib/insert_refunds.rb

```diff
@@ -70,7 +70,7 @@ def self.modern_refund(charge,h)
     
       JobQueue.queue JobTypes::RefundCreatedJob, refund
     
-      return {'payment' => payment.attributes, 'refund' => refund.attributes}
+      {'payment' => payment.attributes, 'refund' => refund.attributes}
```

Comment:
Using a return inside a transaction block is deprecated and will result in a rollback in Rails 7